Batch observer (#1137)
open-telemetry/opentelemetry-js: packages/opentelemetry-metrics/README.md

```

### Value Observer
Choose this kind of metric when only last value is important without worry about aggregation

```js
const { MeterProvider } = require('@opentelemetry/metrics');

const meter = new MeterProvider().getMeter('your-meter-name');

meter.createValueObserver('cpu_core_usage', {
  monotonic: false,
  description: 'Example of a sync observer with callback',
}, (observerResult) => {
  observerResult.observe(getRandomValue(), { core: '1' });
  observerResult.observe(getRandomValue(), { core: '2' });
});

function getRandomValue() {
  return Math.random();
}

```

### Batch Observer

> Choose this kind of metric when you need to have an async calculation that needs to be used to update Value Observer

Reviewer: ```suggestion
Choose this kind of metric when you need to update multiple observers with the results of a single async calculation.
```
